src/client/Client.h

```diff
@@ -730,6 +729,7 @@ class Client : public Dispatcher, public md_config_obs_t {
   void flush_mdlog_sync();
   void flush_mdlog(MetaSession *session);
   
+  xlist<Inode*> get_dirty_list() { return dirty_list; }
```

Comment:
should be `xlist<Inode *> &get_dirt_list()...`

Reply:
thanks you
updated..